pkg/olm/installplanlist.go

```diff
@@ -0,0 +1,45 @@
+package olm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListInstallPlan returns installplan (first from the list).
```

Comment:
Please edit function comment. This function returns just list of InstallPlanBuilder

Reply:
oh yes, sorry, leftover from refactoring